extensions/mssql/src/objectExplorerNodeProvider/treeNodes.ts

```diff
@@ -69,7 +69,7 @@ export abstract class TreeNode implements ITreeNode {
 	/**
 	 * The value to use for this node in the node path
 	 */
-	public abstract get nodePathValue(): string;
+	public abstract get nodePath(): string;
```

Comment:
Ah, here's the root of the issue. Please undo this 😄

Reply:
fixed.